server/src/test/java/org/cloudfoundry/identity/uaa/oauth/jwt/JwtHelperTest.java

```diff
@@ -31,4 +31,10 @@ public void jwtHeaderShouldContainJkuInTheHeader() {
         Jwt jwt = JwtHelper.encode("testJwtContent", keyInfo);
         assertThat(jwt.getHeader().getJku(), is("https://localhost/uaa/token_keys"));
     }
+
+    @Test
+    public void jwtHeaderShouldContainJkuInTheHeaderEvenWithX5t() {
+        Jwt jwt = JwtHelper.encode("testJwtContent", keyInfo, true);
+        assertThat(jwt.getHeader().getJku(), is("https://localhost/uaa/token_keys"));
```

Comment:
Shouldn't we try to use only one assert format consistently throughout the codebase?

i.e. either use the `assertEquals(x,y)` or this `assertThat(x, is(y))`

Reply:
This part is outdated

Reply:
@strehle and I confirmed this is adjusted.